content/events/2019-02-02-project-hub.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "LP x CSSS: Project Hub ft. Intro to Machine Learning"
+date: 2018-02-02
```

Comment:
This is the publish date, set it to today or the post won't appear until Saturday